custom-resource/index.py

```diff
@@ -442,6 +444,15 @@ def create(self):
                 logGroupName=log_group
             )
 
+    @handle_exceptions
```

Comment:
`handle_exceptions` isn't needed here since you explicitly check the length of the function before indexing it, there are no opportunities for exceptions to occur.